docs/management/admin/response-actions.asciidoc

```diff
@@ -105,20 +108,20 @@ TIP: You can also get a list of commands in the <<help-panel,Help panel>>, which
 
 Click image:images/help-icon.png[Help icon,17,18] *Help* in the upper-right to open the *Help* panel, which lists available response action commands and parameters as a reference.
 
+[role="screenshot"]
+image::images/response-console-help-panel.png[Help panel,50%]
+
 You can use this panel to build commands with less typing. Click the add icon (image:images/add-command-icon.png[Add icon,17,17]) to add a command to the input area, enter any additional parameters or a comment, then press *Return* to run the command.
 
+If the endpoint is running an older version of {agent}, some response actions may not be supported, as indicated by an informational icon and tooltip. Upgrade {agent} on the endpoint to add support for the latest response actions.
```

Comment:
This might be a good spot to link to the Agent upgrade docs.

Reply:
Done in https://github.com/elastic/security-docs/pull/2536/commits/2888e4ec5e9842d11598eb50e3bc95032686707d